shared/feature/feedings/src/main/java/com/epmedu/animeal/feedings/presentation/viewmodel/FeedingsViewModel.kt

```diff
@@ -79,14 +80,15 @@ internal class FeedingsViewModel @Inject constructor(
         feedingPoint: FeedingPoint
     ): FeedingModel? {
         val feedingStatus = feeding.status.toFeedingModelStatus(
-            deltaTime = System.currentTimeMillis() - feeding.date.time
+            deltaTime = System.currentTimeMillis() - feeding.date.time,
+            isFeederTrusted = feeding.feeder?.isTrusted == true
         )
 
         return if (feeding.feedingPointId == feedingPoint.id && feedingStatus != null) {
             FeedingModel(
                 id = feeding.id,
                 title = feedingPoint.title,
-                feeder = "${feeding.name} ${feeding.surname}",
+                feeder = "${feeding.feeder?.name.orEmpty()} ${feeding.feeder?.surname.orEmpty()}",
```

Comment:
What about copying the whole model instead of just name?

Reply:
I'm not sure, it has the `isTrusted` field that I wouldn't want to expose to UI